src/main/java/step2/common/ArithmethicConst.java

```diff
@@ -0,0 +1,12 @@
+package step2.common;
+
+public class ArithmethicConst {
```

Comment:
불필요한 약어의 사용을 지양하여 주시면 좋겠네요. 
```suggestion
public class ArithmethicConstants {

Reply:
지금까지는 클래스명이 너무 길어지는듯하여 약어를 사용하곤했었는데
앞으로는 불필요한 약어는 지향하는 것으로 습관을 고치겠습니다 ^^